src/screens/Home/Home.js

```diff
@@ -58,44 +88,111 @@ export function Home({ navigation }) {
   }
 
   const Movie = ({ item }) => (
-    <View>
+    <View style={styles.container_items_movies}>
```

Comment:
Please keep consistency with cases. In this template, we use `camelCase`. Also, let me suggest you another name for this container (a little shorter name).
```suggestion
    <View style={styles.moviesContainer}>
```
(`movieListContainer` is also good)

Reply:
Done!